RiotSwiftUI/Modules/Room/VoiceBroadcastPlayback/MatrixSDK/VoiceBroadcastPlaybackViewModel.swift

```diff
@@ -164,6 +170,52 @@ class VoiceBroadcastPlaybackViewModel: VoiceBroadcastPlaybackViewModelType, Voic
         audioPlayer?.stop()
     }
     
+    /// Backward (30sec) a voice broadcast
+    private func backward() {
+        let newProgressValue = context.progress - VoiceBroadcastPlaybackViewModel.defaultBackwardForwardValue
+        seekTo(newProgressValue > 0 ? newProgressValue : 0)
+        
+        state.bindings.progress = newProgressValue
+        updateBackwardForwardButtons()
+    }
+
+    /// Forward (30sec) a voice broadcast
+    private func forward() {
+        let newProgressValue = context.progress + VoiceBroadcastPlaybackViewModel.defaultBackwardForwardValue
+        seekTo(newProgressValue < state.playingState.duration ? newProgressValue : state.playingState.duration)
+
+        state.bindings.progress = newProgressValue
+        updateBackwardForwardButtons()
+    }
```

Comment:
It seems to me possible to combine forward and backward methods.
https://stackoverflow.com/a/46417693